src/components/Contact/Help.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StyleSheet, Text, View, TouchableOpacity, Linking } from 'react-native';
+
+export default class Header extends React.Component {
+  render() {
+    return (
+      <TouchableOpacity onPress={() => Linking.openURL('https://www.surveymonkey.com/r/Y6JCLQG')}>
+        <Text style={styles.help}>
+          Take a minute to help.
+        </Text>
+      </TouchableOpacity>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  help: {
+    color: 'white',
+    fontSize: 20,
+    textAlign: 'center',
+    textDecorationLine: 'underline',
+    verticalAlign: 'middle',
+    top: '50%',
+  }
+});
```

Comment:
Contact/Help.js: Good use of `TouchableOpacity onPress` and `Linking.openURL` to enable visitors to respond to a survey. Code is clean and well-organized for future additions/amendments.